styles/containers.css

```diff
@@ -161,14 +190,14 @@ hr{
     padding-bottom: 1rem;
 }
 
-.card-code-box{
+.card-code-box {
     margin-top: 2rem;
     margin-left: 8rem !important;
 }
 
 /* Images */
 
-.image-container{
+.image-container {
```

Comment:
the images are not responsive, you can give height auto so that the aspect ratio will remain same.

Reply:
Thanks for pointing, I haven't made the website responsive yet. Working on it.